pkg/controller/logstorage/users/user_controller.go

```diff
@@ -172,13 +174,30 @@ func (r *UserController) Reconcile(ctx context.Context, request reconcile.Reques
 		return reconcile.Result{}, nil
 	}
 
+	// TODO: Fetch cluster UUID and use it in construction of ES usernames so that we can determine which cluster
+	// is responsible for which ES users in Calico Cloud deployments where multiple multi-tenant management clusters
+	// are connecting to a single ElasticSearch instance.
+	clusterIDConfigMap := corev1.ConfigMap{
+		ObjectMeta: apiv1.ObjectMeta{
+			Name:      "cluster-info",
+			Namespace: "tigera-operator",
+		},
+	}
+	err = r.client.Get(ctx, client.ObjectKey{Name: "cluster-info", Namespace: "tigera-operator"}, &clusterIDConfigMap)
+	if err != nil {
+		r.status.SetDegraded(operatorv1.ResourceReadError, "Waiting for cluster-info configmap to be available", nil, reqLogger)
```

Comment:
user-facing error messages should always include both the name and namespace of the resource.

So this should be "Waiting for configmap tigera-operator/cluster-info to be available"